### ISSUE TITLE
Forgot Username

### ISSUE THREAD

**Issue by GiessC** [GiessC/sweep#47]
We have a forgot username page, but it holds no functionality. We need to implement the functionality, but it is out-of-scope for the current Cognito + JWT PR.

Requirements:
- Page must have a form
- Form must have an email field
- Upon submission:
  - Query database for username with given email
  - If username exists, send email containing the username

This will require infrastructure additions, since we will need to setup AWS SES. SES should also be attached to the existing Cognito user pool to replace the sketchy email we send our verification codes through.

Depends on #37
